test_scripts/Polices/build_options/032_ATF_P_TC_PTU_Restarting_Policy_Table_Exchange_HTTP.lua

```diff
@@ -254,11 +257,35 @@ local function DelayedExp(time)
           maxNumberRFCOMMPorts = 1
         }
       })
-    EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "LOCK_SCREEN_ICON_URL"}, {requestType = "HTTP"}):Times(2)
+
+    self.mobileSession2:ExpectNotification("OnSystemRequest"):Times(Between(1,2))
+    :Do(function(_,data)
+        print("SDL->MOB2: OnSystemRequest, requestType: " .. data.payload.requestType)
+        if(data.payload.requestType == "HTTP") then
+          onsysrequest_app2 = true
+          if(onsysrequest_app1 == true) then self:FailTestCase("OnSystemRequest(HTTP) for application 1 already received") end
+        end
+      end)
+
+    self.mobileSession:ExpectNotification("OnSystemRequest"):Times(Between(0,1))
+    :Do(function(_,data)
+        print("SDL->MOB1: OnSystemRequest, requestType: " .. data.payload.requestType)
+        if(data.payload.requestType == "HTTP") then
+          onsysrequest_app1 = true
+          if(onsysrequest_app2 == true) then self:FailTestCase("OnSystemRequest(HTTP) for application 2 already received") end
+        end
+      end)
+
```

Comment:
@istoimenova, Please add DelayedExp() function since Times(Between(0,1)) is used and in this case minimum occurrences = 0, so ATF won't wait for the event 
In case of Times(Between(1,2)) ATF will wait only for the 1st occurrence of event